File: preview/understand/task/create-quickstart/create-quickstart.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$task = $twilio->preview->understand->assistants("UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX")
                                    ->tasks
                                    ->create("tell-a-joke", // uniqueName
                                             array(
                                                 "friendlyName" => "tell a joke",
                                                 "actions" => array(
                                                     "actions" => array(
                                                         array(
                                                             "say" => array(
                                                                 "speech" => "I was going to look for my missing watch, but I could never find the time."
                                                             )
                                                         )
                                                     )
                                                 )
                                             )
                                    );

print($task->sid);

File: autopilot/v1/task/create-quickstart/create-quickstart.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Autopilot.V1.Assistant;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var task = TaskResource.Create(
            friendlyName: "tell a joke",
            uniqueName: "tell-a-joke",
            pathAssistantSid: "UAXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
        );

        Console.WriteLine(task.Sid);
    }
}
